<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BazDan</class>
 <widget class="QDialog" name="BazDan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>368</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>594</width>
    <height>368</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>511</width>
     <height>321</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>511</width>
     <height>321</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>90</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="BaltFlot">
      <property name="text">
       <string>BaltFlot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="KaspFlot">
      <property name="text">
       <string>KaspFlot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TihoFlot">
      <property name="text">
       <string>TihoFlot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="VMF_Rus">
      <property name="text">
       <string>VMF_Rus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>319</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
